roles/os_env_extras_node/tasks/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+- copy: src=enter-container.sh dest=/usr/local/bin/enter-container.sh mode=0755
```

Comment:
we should be using 'docker exec' and not enter-container. The reason for this is selinux. When you use nsenter, and change things inside of the running container, container processes will start getting selinux denials when accessing whatever you changed.

This script looks like one I wrote awhile ago. If this is mine, I think we should just nuke the script entirely.